cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import collections
+import pytest
+
+from cfme.common.physical_server_views import PhysicalServersView
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="function")
+def physical_servers(appliance, provider, setup_provider):
+    # Get and return all the physical servers
+    return appliance.collections.physical_servers.all(provider)
+
+
+@pytest.fixture(scope="function")
+def physical_servers_collection(appliance):
+    # Get and return collection of the physical servers
+    return appliance.collections.physical_servers
+
+
+# Configuration Button
+def test_refresh_relationships(physical_servers_collection, physical_servers, provider):
+    view = navigate_to(physical_servers_collection, "All")
+    last_refresh = provider.last_refresh_date()
+    item = "Refresh Relationships and Power States"
+    physical_servers_collection.custom_button_action("Configuration", item, physical_servers)
+    out, time = wait_for(
+        lambda: last_refresh != provider.last_refresh_date(),
+        fail_func=view.browser.refresh,
+        message="Wait for the servers to be refreshed...",
+        num_sec=300,
+        delay=5
+    )
+    assert out
+
+
+Action = collections.namedtuple('Action', 'button item method')
```

Comment:
This use of a space-delineated string is new to me, but heck, it works!

Reply:
yap, it works